components/content-service/pkg/git/git.go

```diff
@@ -338,8 +342,18 @@ func (c *Client) Clone(ctx context.Context) (err error) {
 		log.WithError(err).Error("cannot create clone location")
 	}
 
+	now := time.Now()
+
+	defer func() {
+		log.WithField("duration", time.Since(now).String()).WithField("FullClone", c.FullClone).Info("clone repository took")
+	}()
+
 	args := []string{"--depth=1", "--shallow-submodules", c.RemoteURI}
 
+	if c.FullClone {
+		args = []string{c.RemoteURI}
+	}
```

Comment:
nit: here I think it'd be more readable to do the inverse (i.e. start with `c.RemoteURI` and check `!c.FullClone`).

Reply:
I would prefer to keep old logic first

Reply:
Am fine with both, resolving